src/components/slider/slider.scss

```diff
@@ -0,0 +1,120 @@
+@import 'default-theme';
+
+$slider-size: 20px !default;
+$slider-min-size: 128px !default;
+
+$slider-track-height: 2px !default;
+$slider-thumb-width: 20px !default;
+$slider-thumb-height: $slider-thumb-width !default;
+
+$slider-thumb-default-scale: 0.7 !default;
+$slider-thumb-focus-scale: 1 !default;
+
+// TODO(iveysaur): Find an implementation to hide the track under a disabled thumb.
+$slider-off-color: rgba(black, 0.26);
+$slider-focused-color: rgba(black, 0.38);
+$slider-disabled-color: rgba(black, 0.26);
+
+@mixin slider-thumb-position($width: $slider-thumb-width, $height: $slider-thumb-height) {
```

Comment:
This only seems to be used in one place (with the default values).  Is this necessary?